.github/workflows/main.yaml

```diff
@@ -83,6 +79,10 @@ jobs:
         make configure-bom
         make test
 
+    - name: Verify
```

Comment:
are tests supposed to produce any uncommitted files ?

We should move the verify step right after build to fail early rather than failing after completing tests.

Reply:
Yeah, fair point. Go used to produce go.mod changes even during tests, but it's no longer the case. Will move this to right after build step.